scout/server/blueprints/dashboard/controllers.py

```diff
@@ -43,20 +43,41 @@ def get_dashboard_info(adapter, institute_id=None, slice_query=None):
     data['analysis_types'] = get_analysis_types(adapter, total_cases,
                                 institute_id=institute_id, slice_query=slice_query)
 
+    # VARIANTS
+
+    # Here we disregard any slice query.
+    general_info = get_general_case_info(adapter, institute_id=institute_id)
```

Comment:
Question: why do you want to disregard the filter from here? Then it will happen that some stats will be based on all cases available and some other on the slice_query. Wouldn't it be messy then to understand where the numbers come from?

Reply:
Yes, that is non-obvious. Personally, I don't see the use for fraction of verifying variants in any of the current slice categories. If we have subsets like "low quality variants", "tissue", "cancer" or such it suddenly makes a lot of sense!

Couldn't we just make two distinct panels on the page: one with cases, which is sliceable. And then one with variants, which is so far not sliceable, and has the download button grouped with it?

Reply:
It should be quite easy to encapsulate the distinct parts in 2 panels, this way it will be more clear the data in which the slice query has an effect. I can draft this